c7n/resources/elb.py

```diff
@@ -272,12 +274,13 @@ def process_elb(self, elb):
 
         client = local_session(self.manager.session_factory).client('elb')
 
-        # Create a custom policy.
-        attrs = self.data.get('attributes')
-        # This name must be unique within the
+        # Create a custom policy with epoch timestamp.
+        # to make it unique within the
         # set of policies for this load balancer.
-        policy_name = self.data.get('name')
+        policy_name = self.data.get('name') + '-' + \
+                      str(int(datetime.now(tz=tzutc()).strftime("%s")) * 1000)
```

Comment:
this seems a bit curious, your effectively adding multiple ssl load balancer policies here. what's the actual behavior effect on the elb, and why do this vs replace?